packages/workbox-background-sync/src/lib/StorableRequest.ts

```diff
@@ -22,6 +25,13 @@ const serializableProperties = [
   'keepalive',
 ];
 
+export interface RequestData {
+  url: string;
+  headers: {[headerName: string]: any};
```

Comment:
Should this be typed as that official `Headers` interface you manually added? Or is this not actually a real `Headers`?

Reply:
It's an object of key/value pairs. I suppose it may make more sense to rework this as a `HeadersInit` object (i.e. `string[][]`) and reuse the existing typing, but yeah, it's definitely not a `Headers` object because it needs to be structured cloneable.

Reply:
Gotcha.